modules/light-clients/07-tendermint/types/update_test.go

```diff
@@ -27,7 +27,7 @@ func (suite *TendermintTestSuite) TestCheckHeaderAndUpdateState() {
 
 	// Setup different validators and signers for testing different types of updates
 	altPrivVal := ibctestingmock.NewPV()
-	altPubKey, err := altPrivVal.GetPubKey()
+	altPubKey, err := altPrivVal.GetPubKey(suite.ctx)
```

Comment:
```suggestion
	altPubKey, err := altPrivVal.GetPubKey(suite.chainA.GetContext())
```
preferably `suite.ctx` is removed in the future

or pass in "nil" if possible since the context is discarded anyways, or `context.Background()`